index.js

```diff
@@ -115,7 +115,7 @@ export default function typescriptProvider({negotiateProtocol}) {
 						return false;
 					}
 
-					return rewritePaths.some(([from]) => filePath.startsWith(from));
+					return rewritePaths.some(([to, from]) => filePath.startsWith(compile === 'tsc' ? from : to));
```

Comment:
Would it make sense to check `compile` once? Also you mixed up the "from" and "to":

```suggestion
					return compile === 'tsc' ? rewritePaths.some(([from, to]) => filePath.startsWith(to)) : rewritePaths.some(([from]) => filePath.startsWith(from));
```

Reply:
lol. yep, sorry